.idea/externalDependencies.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ExternalDependencies">
+    <plugin id="org.squareup.cash.hermit.idea-plugin" />
+  </component>
+</project>
```

Comment:
@alecthomas Should this go in `files/.idea/externalDependencies.xml`?

Reply:
Yeah, though I think just `files/externalDependencies.xml` is okay.